src/services/JobRoleService.ts

```diff
@@ -1,5 +1,6 @@
+import axios, {AxiosResponse} from "axios"
+import { JobRole } from "../models/JobRole"
```

Comment:
Import as a type to save the polar bears

Reply:
Done
<img width="662" alt="image" src="https://github.com/user-attachments/assets/40e2d3ef-9058-45d1-b854-404439391b3f">